frontend/src/components/common/modal/styles.module.scss

```diff
@@ -0,0 +1,49 @@
+.modalBackgroundContainer {
+  position: fixed;
+  z-index: 100;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+  height: 100%;
+  background-color: #1e1e2cb5;
+}
+
+.modalContainer {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+  height: 80%;
+  margin: auto;
+  color: white;
+  background-color: var(--background-gray-400);
+}
+
+.modalClose {
+  width: 20px;
+  margin-right: 20px;
+  margin-left: auto;
+  padding-top: 5px;
+  color: var(--background-gray-200);
+  font-size: 25px;
```

Comment:
For more adaptive, mb it's better to use **rem**
25px = 1.5625rem

Reply:
No, in hook linter rem is forbidden :(

Reply:
Ok(